src/app/services/http.request.interceptor.ts

```diff
@@ -27,10 +29,14 @@ export class HttpsRequestInterceptor implements HttpInterceptor {
         return event;
       }), catchError((err: any) => {
         if (err instanceof HttpErrorResponse) {
-          if (err.status === 401 && !location.href.includes('/setting?id=1')) {
+          if (err.status === 0) {
+            this.alertService.error('Failed to connect');
+          } else if (err.status === 401 && !location.href.includes('/setting?id=1')) {
             sessionStorage.clear();
             this.router.navigate(['/login']);
```

Comment:
this.servicesRecord = []; and replaceUrl: true  not needed here? as we have in clearUserSession func?

Reply:
`servicesRecord` doesn't exist on this page but `replaceUrl: true` can be used here.